src/Amazon.Extensions.CognitoAuthentication/CognitoUser.cs

```diff
@@ -373,6 +374,20 @@ public virtual async Task<List<CognitoDevice>> ListDevicesAsync(int limit, strin
             return devicesList;
         }
 
+        /// <summary>
+        /// Executes the ListDevicesAsync service call to access device types associated with this user using an asynchronous call. 
+        /// The response returned contains DeviceType objects which could be used to construct list of CognitoDevice type objects and 
+        /// a PaginationToken which could be used to access remaining device types (if any).
+        /// </summary>
+        /// <param name="limit">Maxmimum number of devices to be returned in this call</param>
+        /// <param name="paginationToken">Token to continue earlier search</param>
+        /// <returns>Returns underlying ListDevicesResponse that contains list of DeviceType objects along with PaginationToken.</returns>
+        public virtual async Task<ListDevicesResponse> ListDevicesWithPaginationTokenAsync(int limit, string paginationToken)

```

Comment:
Not a fan of the name `ListDevicesWithPaginationTokenAsync`. I think I would actually prefer `ListDevicesV2Async` so users understand these methods are intended to do same thing but fix the issues of the first one. Adding parameter name to the method name seems uglier.

Reply:
Fixed in latest commit.